ios/RNGoogleSignin/RNGoogleSignin.m

```diff
@@ -97,24 +161,24 @@ - (void)signIn:(GIDSignIn *)signIn didSignInForUser:(GIDGoogleUser *)user withEr
                            @"accessTokenExpirationDate": [NSNumber numberWithDouble:user.authentication.accessTokenExpirationDate.timeIntervalSinceNow]
                            };
 
-    return [self.bridge.eventDispatcher sendAppEventWithName:@"RNGoogleSignInSuccess" body:body];
-}
+    resolve(body);
```

Comment:
`resolve` is defined on line 115, it would make the code more readable if it was defined right here

Reply:
You’re right.